server/src/main/java/org/gluu/oxtrust/ldap/service/AppInitializer.java

```diff
@@ -253,12 +258,26 @@ public PersistenceEntryManager createLdapEntryManager() {
         EncryptionService securityService = encryptionServiceInstance.get();
         Properties decryptedConnectionProperties = securityService.decryptProperties(connectionProperties);
 
-        PersistenceEntryManager ldapEntryManager = this.ldapEntryManagerFactory.createEntryManager(decryptedConnectionProperties); 
+        PersistenceEntryManager ldapEntryManager = this.ldapEntryManagerFactory.createEntryManager(decryptedConnectionProperties);
         log.info("Created {}: {}", new Object[] { LDAP_ENTRY_MANAGER_NAME, ldapEntryManager.getOperationService() });
 
         return ldapEntryManager;
     }
 
+    @Produces
```

Comment:
I think we can avoid this `@Produces`. Let's review OxAuthConfigurationDAO  according comments above